samcli/commands/pipeline/bootstrap/guided_context.py

```diff
@@ -89,7 +89,7 @@ def _prompt_stage_configuration_name(self) -> None:
             "This will be referenced later when you use the sam pipeline init command:"
         )
         self.stage_configuration_name = click.prompt(
-            "Configuration name",
+            "Stage configuration name",
```

Comment:
It's called "stage configuration name" here, but in many places just "configuration name" in https://github.com/aws/aws-sam-cli/pull/3425. "Configuration" is such an overloaded term we should probably make them all consistent.